### PR TITLE
Added gitter badge / Fixed security vulnerabilities

### PR DESCRIPTION
**Pull request by kotielnikov**
### Prerequisites

Please make sure you can check the following two boxes:

- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
- [x] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My change requires a change to the documentation
- [x] I have updated the documentation accordingly
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Precise UI Changelog
 
+## 0.7.0
```
Comment (FlorianRappl) [inline CHANGELOG.md]: Is there a breaking change? From the changelog it does not seem like it.

Comment (kotielnikov) [reply CHANGELOG.md]: why? I updated the minor version, not major. PR brings a new functional - gitter badge which corresponds to minor version description

> MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards-compatible manner, and
PATCH version when you make backwards-compatible bug fixes.

Comment (FlorianRappl) [reply CHANGELOG.md]: This is only for 1.0.0 onwords; for pre v1 semver says you may treat the minor as major etc. (i.e., the first significant digit is the major, rest follows). This has been done, e.g., by React and is a quite successful system.

Further reading on pre v1 semver: https://github.com/semver/semver/issues/333

Comment (FlorianRappl) [reply CHANGELOG.md]: BTW: The Gitter badge is not a functional addition; this is a README change and has no functional impact.

Comment (kotielnikov) [reply CHANGELOG.md]: Ok, didn't know that. Thanks.
Fixed

**Pull request merged**